core/storage/user/gae_models.py

```diff
@@ -96,6 +96,11 @@ class UserEmailPreferencesModel(base_models.BaseModel):
     # None if the user has never set a preference.
     site_updates = ndb.BooleanProperty(indexed=True)
 
+    # The user's preference for receiving email when user is added as a member
+    # in exploration. This is set to True when user has never set a preference.
+    membership_notifications = ndb.BooleanProperty(
```

Comment:
I've been thinking about this, and I think a better name would be editor_role_notifications. I don't think we currently have the concept of "member", or that we want to introduce the new terminology yet; I found 'role' in core/domain/rights_manager.py. What do you think?